packages/rum/src/domain/record/observers.ts

```diff
@@ -96,10 +97,16 @@ interface ObserverParam {
   styleSheetCb: StyleSheetCallback
   focusCb: FocusCallback
   frustrationCb: FrustrationCallback
+  shadowDomCreatedCb: ShadowDomCallBack
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏all other callbacks are used to emit a record. It feels a bit strange to have this one at the same level.

Reply:
I do not have a strong opinion on this, if you provide a suggestion, I will implement it